src/database/UIDatabase.js

```diff
@@ -13,10 +13,14 @@ export class UIDatabase {
       case 'Supplier':
         results = results.filtered('isVisible == true AND isSupplier == true');
         break;
+      case 'mSupplyStore':
```

Comment:
Yes indeed kudos. I think all cases should be returns, don't see how break-ing every case is helpful here